lib/serializer.ts

```diff
@@ -95,14 +119,24 @@ function symbolCompareFunction(a: ts.Symbol, b: ts.Symbol) {
 /**
  * Traverses the node tree to construct the text without comments and privates.
  */
-function getSanitizedText(node: ts.Node): string {
+function getSanitizedText(node: ts.Node, options: SerializationOptions): string {
   if (node.flags & ts.NodeFlags.Private) {
     return '';
   }
 
+  const firstQualifier: ts.Identifier = getFirstQualifier(node);
+
+  if (firstQualifier) {
+    if (!options.allowModuleIdentifiers ||
+        options.allowModuleIdentifiers.indexOf(firstQualifier.text) < 0) {
+      throw new Error(createErrorMessage(
+          firstQualifier, `Module identifier "${firstQualifier.text}" is not whitelisted`));
```

Comment:
... is not allowed. Remove it from source or whitelist via --allowModuleIdentifiers